plugins/base/src/main/kotlin/DokkaBase.kt

```diff
@@ -33,6 +32,7 @@ import org.jetbrains.dokka.plugability.PluginApiPreviewAcknowledgement
 import org.jetbrains.dokka.transformers.documentation.PreMergeDocumentableTransformer
 import org.jetbrains.dokka.transformers.pages.PageTransformer
 
+@Suppress("unused")
```

Comment:
I see that the suppress migrated from file-level to class-level, which makes it more noticeable and questionable. Is it even needed? Maybe let's remove it completely? As far as I know, the compiler/ide shouldn't complain about unused public API

Reply:
> As far as I know, the compiler/ide shouldn't complain about unused public API

This inspection is enabled in IDE by default (in Kotlin plugin).

Reply:
It's enabled, and the unused properties are highlighted in gray, but neither IDEA nor the compiler seem to complain about it, i.e underline them with wiggly lines or fail the build due to `enableWarningsAsErrors`. So to me this suppress does nothing useful, but can backfire in the future. 

But it's up to you, I don't think it's important

Reply:
![image](https://github.com/Kotlin/dokka/assets/9636937/5658e581-43bc-4e59-8c07-929048700477)